### PR TITLE
add sinc etc.

### PR DESCRIPTION
**Pull request by zpceng314**
### PR APIs

torch.hypot
torch.Tensor.hypot
torch.Tensor.histc
torch.Tensor.istft
torch.Tensor.logder
torch.Tensor.negative
torch.Tensor.pinverse
torch.Tensor.reshape_as
torch.Tensor.select
torch.select
torch.Tensor.sinc
torch.special.sinc
torch.sinc
torch.random.get_rng_state
torch.random.set_rng_state
torch.randomseed
torch.slogdet
torch.linalg.slogdet

### PR Docs

https://github.com/PaddlePaddle/docs/pull/5894
### Description
...

### CODE REVIEW

File: paconvert/api_matcher.py [hunk 1]
```diff
@@ -3509,6 +3524,85 @@ def get_paddle_class_nodes(self, func, args, kwargs):
...
+            self.paddleClass, kwargs["index"], kwargs["dim"], kwargs["dim"]
+        )
+
+        return ast.parse(code).body
+
+
+class SincMatcher(BaseMatcher):
+    def generate_code(self, kwargs):
+
+        API_TEMPLATE = textwrap.dedent(
```
[the remark this answers is not in the document]
Comment (zpceng314) [reply paconvert/api_matcher.py]: done!

File: paconvert/api_mapping.json [hunk 2]
```diff
@@ -524,7 +537,6 @@
       "end",
       "steps",
       "base",
-      "out",
```
[the remark this answers is not in the document]
Comment (zpceng314) [reply paconvert/api_mapping.json]: done!

File: paconvert/api_mapping.json [hunk 3]
```diff
@@ -955,12 +976,11 @@
     "paddle_api": "paddle.randperm",
     "args_list": [
       "n",
-      "generator",
```
[the remark this answers is not in the document]
Comment (zpceng314) [reply paconvert/api_mapping.json]: done!

File: paconvert/api_mapping.json [hunk 4]
```diff
@@ -2033,7 +2057,8 @@
       "out"
     ],
     "kwargs_change": {
```
[the remark this answers is not in the document]
Comment (zpceng314) [reply paconvert/api_mapping.json]: done! 可能是分支合并，解决冲突不正确造成的

File: paconvert/api_matcher.py [hunk 5]
```diff
@@ -3509,6 +3524,99 @@ def get_paddle_class_nodes(self, func, args, kwargs):
         return ast.parse(code).body
 
 
+class TensorPinverseMatcher(BaseMatcher):
+    def get_paddle_class_nodes(self, func, args, kwargs):
```
Comment (zhwesky2010) [inline paconvert/api_matcher.py:3528]: 这些没有可变参数的，重写 `generate_code` 就可以，更简单点，其他API也都看看这个问题。尽量还是重写  `generate_code`

Comment (zpceng314) [reply paconvert/api_matcher.py]: done!

**Review by zhwesky2010 — APPROVED** [review]

**Pull request merged**